tcases-lib/src/main/java/org/cornutum/tcases/VarBindingDef.java

```diff
@@ -99,7 +99,7 @@ public ICondition getEffectiveCondition()
    */
   public boolean isNA()
     {
-    return VarValueDef.isNA( valueDef_);
+    return valueDef_ != null && valueDef_.isNA();
```

Comment:
this looks like there are now 2 ways to create NA values in the code (maybe not for XML users). That seems bug-prone.